### ISSUE TITLE
Does this repo still compile?

### ISSUE THREAD

**Issue by dentiny** [josephyaconelli/crazyflie-firmware-pwm-control#1]
When I compiled with "make PLATFORM=cf2", it fails with the following message:

/tmp/cc1zmOiG.s: Assembler messages:
/tmp/cc1zmOiG.s:2818: Error: selected processor does not support `clz r4,r4' in ARM mode

I am trying to do some control involving individual motor control with crazyflie 2.x (likely 2.1). Appreciate any help.

**Comment by natolambert** [comment]
What platform are you trying to compile on? Bitcraze virtual machine? 
We noticed some weird issues with the online versions of the firmware and the most recent  virtual machine.

**Comment by dentiny** [comment]
I was compiling inside a ubuntu 16.04 vm in virtualbox running on a 64-bit windows host. I have actually just tried on a ubuntu 16.04 machine locally. Same error about the instruction "clz r4, r4" not supported. I am pretty sure that I was using gcc-arm-embedded package for compilation. Maybe something should be changed in the makefile?

In my ubuntu vm, I was able to compile the master branch from the original bitcraze/crazyflie-firmware. I was planning to compile this firmware locally and flash with the crazyflie PA, (following the instructions from the original repo). Or should I compile and upload firmware differently with this repo?

Thanks a lot for your help on looking into this.
